cdap-ui/app/directives/dag-plus/my-dag.html

```diff
@@ -95,7 +95,7 @@
 
 <div class="my-js-dag"
     ng-class="{'disabled': DAGPlusPlusCtrl.isDisabled, 'normal-cursor': disableNodeClick, 'preview-mode': previewMode }"
-    ng-click="DAGPlusPlusCtrl.clearNodeSelection()">
+    ng-click="DAGPlusPlusCtrl.toggleNodeMenu(null); DAGPlusPlusCtrl.openDagMenu(false);">
```

Comment:
I believe calling it without any param should do the same thing. `nodeName` would be undefined which is a falsy value. Not sure if I am missing something here.